src/stories/components/TabBar.stories.tsx

```diff
@@ -5,13 +5,6 @@ import TabBar from '@/components/TabBar';
 const meta: Meta<typeof TabBar.Container> = {
   title: 'common/TabBar',
```

Comment:
이제 봤는데 이것도 같이 수정해서 머지해주시면 좋을 것 같습니다 ㅎㅎ
```suggestion
  title: 'components/TabBar',
```

Reply:
반영 완료 했습니다 :)